utilities/common.py

```diff
@@ -72,7 +72,7 @@
 ## 10% quantiles from aMC@NLO used in SMP-18-012 with some rounding <== This one worked fine with toys
 ptV_10quantiles_binning = [0.0, 2.95, 4.73, 6.68, 8.98, 11.78, 15.33, 20.11, 27.17, 40.15, 13000.]
 # Integer rounded version of the 5% quantiles h[::hist.rebin(2)] for 10% quantiles
-ptV_binning = [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 15, 18, 20, 23, 27, 32, 40, 54, 13000]
+ptV_binning = [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 15, 17, 20, 23, 27, 32, 40, 54, 13000]
```

Comment:
I guess you can now use this directly in mz_dilepton

Reply:
I think it is better to have a separate definition in mz_dilepton because there is no reason that we have to use the same binning. This is more robust for future changes. Also, the last entry with 13000 is not optimal for the unfolding.